README.md

```diff
@@ -1 +1,9 @@
-#osu-plus
+#  osu-plus
+[![Official International Discord](https://discordapp.com/api/guilds/316545691545501706/widget.png?style=shield)](https://discord.gg/nyD92cE)
+[![Android CI](https://github.com/osudroid/osu-droid/workflows/Android%20CI/badge.svg?branch=master)](https://github.com/osudroid/osu-droid/actions?query=workflow%3A"Android+CI")
+[![CodeFactor](https://www.codefactor.io/repository/github/osudroid/osu-droid/badge)](https://www.codefactor.io/repository/github/osudroid/osu-droid)
+## How to setup!
+```text
+:~$ git clone https://github.com/osudroid/osu-droid.git
+:~$ open it on Android Studio
+```
```

Comment:
You should merge master branch but not use this way to keep update with master branch